### DISCUSSION TITLE
New Feature support for Stable Diffusion Web UI

### DISCUSSION THREAD

**Discussion by devangaggarwal** [openvinotoolkit/stable-diffusion-webui#42] [Ideas]
We are always looking for inputs on how to improve our OpenVINO integration into the Stable Diffusion Web UI.
Provide us with your thoughts and ideas on which features you would like to see in upcoming releases.

**Comment by Micraow** [comment]
Maybe outpainting?

**Comment by tuanakotta** [comment]
why we can't use inpainting checkpoint with openvino script?
